Volume/View Models/ArticleInfo.swift

```diff
@@ -13,18 +13,19 @@ struct ArticleInfo: View {
 
     let article: Article
     let showsPublicationName: Bool
+    let expandedSize: Bool
```

Comment:
I think `large` or `largeFont` or something like that is better. I couldn't tell what `expandedSize` meant just from the usage (without looking in this file)